rotary_irq_rp2.py

```diff
@@ -0,0 +1,67 @@
+# The MIT License (MIT)
+# Copyright (c) 2020 Mike Teachman
+# https://opensource.org/licenses/MIT
+
+# Platform-specific MicroPython code for the rotary encoder module
+# Pico implementation
```

Comment:
`Raspberry Pi Pico`

Reply:
Agreed; I will update the PR code accordingly.